src/commands/setup-for-local-development.js

```diff
@@ -0,0 +1,122 @@
+const Listr = require('listr');
+
+const { flags: flagTypes } = require('@oclif/command');
+
+const BaseCommand = require('../oclif/command/BaseCommand');
+const UpdateRendererWithOutput = require('../oclif/renderer/UpdateRendererWithOutput');
+const MuteOneLineError = require('../oclif/errors/MuteOneLineError');
+
+const PRESETS = require('../presets');
+
+class SetupForLocalDevelopmentCommand extends BaseCommand {
+  /**
+   *
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {generateToAddressTask} generateToAddressTask
+   * @param {registerMasternodeTask} registerMasternodeTask
+   * @param {initTask} initTask
+   * @param {DockerCompose} dockerCompose
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    { port: coreP2pPort, 'external-ip': externalIp },
+    {
+      'drive-image-build-path': driveImageBuildPath,
+      'dapi-image-build-path': dapiImageBuildPath,
+    },
+    generateToAddressTask,
+    registerMasternodeTask,
+    initTask,
+    dockerCompose,
+  ) {
+    const preset = PRESETS.LOCAL;
+    const network = 'testnet';
```

Comment:
Should be "local"

Reply:
There are some other places where we have this value as testnet. Should I change it here too?